app/lib/infrastructure/words_repository.dart

```diff
@@ -0,0 +1,120 @@
+part of 'package:word_link/domain/controllers/words_controller.dart';
+
+class _WordsRepository extends WordsController {
+  static const Map<String, String> polishWords = {
+    'i': 'and',
+    'w': 'in',
+    'nie': 'not',
+    'to': 'it',
+    'że': 'that',
+    'na': 'on',
+    'się': 'self',
+    'z': 'with',
+    'jest': 'is',
+    'do': 'to',
+    'co': 'what',
+    'jak': 'how',
+    'a': 'but',
+    'o': 'about',
+    'tak': 'so',
+    'ale': 'but',
+    'po': 'after',
+    'go': 'him',
+    'mi': 'me',
+    'czy': 'whether',
+    'ty': 'you',
+    'mnie': 'me',
+    'dla': 'for',
+    'już': 'already',
+    'za': 'for',
+    'bo': 'because',
+    'by': 'would',
+    'był': 'was',
+    'być': 'be',
+    'ma': 'has',
+    'tego': 'that',
+    'jej': 'her',
+    'przy': 'near',
+    'ich': 'their',
+    'albo': 'or',
+    'ze': 'with',
+    'mu': 'him',
+    'tym': 'that',
+    'nas': 'us',
+    'też': 'also',
+    'ten': 'this',
+    'od': 'from',
+    'kiedy': 'when',
+    'tam': 'there',
+    'czyli': 'that',
+    'więc': 'so',
+    'jeśli': 'if',
+    'dlatego': 'therefore',
+    'te': 'these',
+    'wszystko': 'everything',
+    'nasz': 'our',
+    'tu': 'here',
+    'trzeba': 'must',
+    'jako': 'as',
+    'bardziej': 'more',
+    'nic': 'nothing',
+    'który': 'which',
+    'było': 'was',
+    'czas': 'time',
+    'gdy': 'when',
+    'jeszcze': 'still',
+    'potem': 'then',
+    'teraz': 'now',
+    'ciebie': 'you',
+    'dobrze': 'well',
+    'raz': 'once',
+    'może': 'maybe',
+    'nigdy': 'never',
+    'nawet': 'even',
+    'wszyscy': 'everyone',
+    'lecz': 'but',
+    'zawsze': 'always',
+    'czemu': 'why',
+    'sam': 'self',
+    'twoje': 'your',
+    'czasu': 'time',
+    'była': 'was',
+    'bardzo': 'very',
+    'sobie': 'self',
+    'są': 'are',
+    'taki': 'such',
+    'tylko': 'only',
+    'nami': 'us',
+    'chce': 'wants',
+    'więcej': 'more',
+    'nasze': 'our',
+    'kto': 'who',
+    'mogę': 'can',
+    'mógł': 'could',
+    'będzie': 'will',
+    'gdzie': 'where',
+    'potrzebuje': 'needs',
+    'czego': 'what',
+    'muszę': 'must',
+    'mój': 'my',
+    'twoja': 'your',
+    'bym': 'would',
+    'kim': 'who',
+    'znowu': 'again',
+  };
+
+
+  @override
+  CollectionObject generateCollection(String name) {
+
+    
+    final keys = polishWords.keys.toList()..shuffle(Random());
+    final selected = keys.take(5);
+    final wordPairs = selected.map((key) => CardObject(name: key, answer: polishWords[key])).toList();
+
+    return CollectionObject(name: name, cardsTemp: wordPairs);
+
+    
```

Comment:
Remove extra spacing.
Make sure your IDE is organizing the code each time you use the "ctrl + s" save shortcut.
It will make your code nice.